.gitignore

```diff
@@ -19,3 +19,6 @@ coldfront.db
 .vscode
 db.json
 .env
+.devcontainer/*
+.gitignore
```

Comment:
I don't think we want to gitignore the gitignore file here :) Can you remove?

Reply:
No we don't. :)